src/jit/gentree.cpp

```diff
@@ -6304,6 +6532,7 @@ GenTreePtr          Compiler::gtCloneExpr(GenTree * tree,
         copy->gtCall.gtInlineObservation = tree->gtCall.gtInlineObservation;
 #endif
 
+        copy->AsCall()->CopyOtherRegFlags(tree->AsCall());
```

Comment:
This is copying of spill flags associated with the individual regs.  That won't get copied as part of CopyReg.  This call is needed here.